### PR TITLE
Add chaospy

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipes/chaospy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  source_files:
+    - tests
+  requires:
+    - pytest-runner
+  commands:
+    - py.test
+
+about:
+  home: https://github.com/jonathf/chaospy
+  license: BSD
```
Comment (dougalsutherland) [inline recipes/chaospy/meta.yaml]: Should be BSD-3-Clause, and `license_family: BSD`.

File: recipes/chaospy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - networkx
+    - scikit-learn
```
Comment (dougalsutherland) [inline recipes/chaospy/meta.yaml]: The `setup.py` also explicitly lists `numpy` and `scipy` as dependencies; they're of course transitively required by `scikit-learn`, but could probably list them for completeness....

File: recipes/chaospy/meta.yaml (hunk 1 above)
Comment (jschueller) [reply recipes/chaospy/meta.yaml]: ok

**Comment by dougalsutherland** [comment]
@jsturdy, are you interested in [being a co-maintainer](http://conda-forge.readthedocs.io/en/latest/recipe.html#maintainer-role) for the conda-forge package of chaospy?

**Comment by jschueller** [comment]
@dougalsutherland I removed him/her as he/she doesn't answer

**Comment by dougalsutherland** [comment]
Merged. Thanks @jschueller!

**Pull request merged**